src/styles/forms.scss

```diff
@@ -8,3 +10,113 @@
     font-size: 1rem;
     padding: 6px 4px;
 }
+
+// Organization -----------------------------------
+
+legend {
```

Comment:
I'd prefer to use classes for our custom form styles rather than specifying tags. Classes make it clearer where applied styling is coming from rather than having it applied "globally".